lib/util/util.ex

```diff
@@ -188,10 +188,10 @@ defmodule Util do
 
   @spec handle_ambiguous_time(Timex.AmbiguousDateTime.t() | DateTime.t() | {:error, any}) ::
           DateTime.t() | {:error, any}
-  defp handle_ambiguous_time(%Timex.AmbiguousDateTime{before: before}) do
+  defp handle_ambiguous_time(%Timex.AmbiguousDateTime{after: aft}) do
```

Comment:
📓 Timex suggests you use after unless you have a reason to use before. `after` is a reserved word hence the use of `aft`.